services/brig/deb/opt/brig/templates/user/de/email/team-activation.html

```diff
@@ -102,7 +103,7 @@
                   color: #B9BCBD;
                   font-weight: 500;
                   text-decoration:none !important;
-                ">Datenschutzrichtlinie</a><span style="margin-left: 8px; margin-right: 8px; color: #B9BCBD;">&middot;</span>
+                ">Datenschutz</a><span style="margin-left: 8px; margin-right: 8px; color: #B9BCBD;">&middot;</span>
```

Comment:
In other changes in this PR, `Datenschutz` was changed to `Datenschutzrichtlinie`. Here, the reverse is done. `Datenschutzrichtlinie` is preferred. Please ensure consistency.